itou/api/siae_api/viewsets.py

```diff
@@ -0,0 +1,129 @@
+import logging
+
+# from django_filters.filters import OrderingFilter
+# from django_filters import FilterSet
+from drf_spectacular.types import OpenApiTypes
+from drf_spectacular.utils import OpenApiExample, OpenApiParameter, extend_schema
+from rest_framework import filters, viewsets
+from rest_framework.exceptions import NotFound, ValidationError
+
+from itou.cities.models import City
+from itou.siaes.models import Siae
+from itou.siaes.serializers import SiaeSerializer
+
+
+logger = logging.getLogger("api_drf")
+CODE_INSEE_PARAM_NAME = "code_insee"
+DISTANCE_FROM_CODE_INSEE_PARAM_NAME = "distance_max_km"
+MAX_DISTANCE_RADIUS_KM = 100
+
+
+class SiaeViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    # Liste des SIAE
+
+    La plateforme renvoie une liste de SIAE à proximité d’une ville (déterminée par son code INSEE)
+    et d’un rayon de recherche en kilomètres autour de cette ville.
```

Comment:
```suggestion
    dans un rayon de recherche en kilomètres autour du centre de cette ville.
```

J'aurais mis un truc comme ça pour bien expliquer qu'on cherche à partir du centre.

Reply:
D’ailleurs (j’ai déjà eu la question, pas su répondre de manière précise), les infos qu’on a (`cities.json`) viennent d’où et comment ils déterminent le centre de la ville ?